task-2/public/styles/scss/blocks/button.scss

```diff
@@ -0,0 +1,26 @@
+.button {
+  &_link {
+    text-decoration: none;
+    @include article-category;
+    &:hover {
+      opacity: 0.4;
+    }
+  }
+  
+  &__text {
+    @include load-button;
+  }
+
+  &_load {
+    align-self: center;
```

Comment:
Why button should align itself? What if you'll need the same button but positioned left or right?